### ISSUE TITLE
Resource not accessible by integration

### ISSUE THREAD

**Issue by jmnote** [pascalgn/size-label-action#33]
"Resource not accessible by integration"
cannot POST label - 403 Forbidden

Here's my workflow file `size-label.yml`
```yaml
name: size-label
on: pull_request
permissions: write-all
jobs:
  size-label:
    runs-on: ubuntu-latest
    steps:
    - name: size-label
      uses: "pascalgn/size-label-action@v0.4.3"
      env:
        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

job log => 403 Forbidden
```
▶ Run pascalgn/size-label-action@v0.4.3
Changed lines: 7
Matching label: size/XS
RequestError [HttpError]: Resource not accessible by integration
    at /home/runner/work/_actions/pascalgn/size-label-action/v0.4.3/dist/index.js:2[5](https://github.com/kuoss/goproject1/actions/runs/3890402527/jobs/6639487264#step:2:6)82:23
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async main (/home/runner/work/_actions/pascalgn/size-label-action/v0.4.3/dist/index.js:97:5) {
  status: 403,
  headers: {
    'access-control-allow-origin': '*',
    'access-control-expose-headers': 'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset',
    connection: 'close',
    'content-encoding': 'gzip',
    'content-security-policy': "default-src 'none'",
    'content-type': 'application/json; charset=utf-8',
    date: 'Wed, 11 Jan 2023 0[6](https://github.com/kuoss/goproject1/actions/runs/3890402527/jobs/6639487264#step:2:7):52:10 GMT',
    'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin',
    server: 'GitHub.com',
    'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
    'transfer-encoding': 'chunked',
    vary: 'Accept-Encoding, Accept, X-Requested-With',
    'x-content-type-options': 'nosniff',
    'x-frame-options': 'deny',
    'x-github-api-version-selected': '2022-11-28',
    'x-github-media-type': 'github.v3; format=json',
    'x-github-request-id': 'A002:6984:386[7](https://github.com/kuoss/goproject1/actions/runs/3890402527/jobs/6639487264#step:2:8)4C[8](https://github.com/kuoss/goproject1/actions/runs/3890402527/jobs/6639487264#step:2:9):735FECC:63BE5C[9](https://github.com/kuoss/goproject1/actions/runs/3890402527/jobs/6639487264#step:2:10)9',
    'x-ratelimit-limit': '[10](https://github.com/kuoss/goproject1/actions/runs/3890402527/jobs/6639487264#step:2:11)00',
    'x-ratelimit-remaining': '993',
    'x-ratelimit-reset': '1673423444',
    'x-ratelimit-resource': 'core',
    'x-ratelimit-used': '7',
    'x-xss-protection': '0'
  },
  request: {
    method: 'POST',
    url: 'https://api.github.com/repos/kuoss/goproject1/issues/9/labels',
    headers: {
      accept: 'application/vnd.github.v3+json',
      'user-agent': 'pascalgn/size-label-action octokit-rest.js/18.5.5 octokit-core.js/3.4.0 Node.js/[12](https://github.com/kuoss/goproject1/actions/runs/3890402527/jobs/6639487264#step:2:13).[22](https://github.com/kuoss/goproject1/actions/runs/3890402527/jobs/6639487264#step:2:23).7 (linux; x64)',
      authorization: 'token [REDACTED]',
      'content-type': 'application/json; charset=utf-8'
    },
    body: '{"labels":["size/XS"]}',
    request: { hook: [Function: bound bound register] }
  },
  documentation_url: 'https://docs.github.com/rest/reference/issues#add-labels-to-an-issue'
}
```

**Comment by jmnote** [comment]
similar issues
- https://github.com/actions/labeler/pull/50
- https://github.com/actions/first-interaction/issues/10

**Comment by pascalgn** [comment]
I merged #34, so I think this is fixed now. However, feel free to reopen this or create a new issue if you disagree or if there are any other problems! 👍
